stellargraph/layer/gcn.py

```diff
@@ -56,47 +56,42 @@ class GraphConvolution(Layer):
         use_bias (bool): toggles an optional bias
         final_layer (bool): If False the layer returns output for all nodes,
                             if True it returns the subset specified by the indices passed to it.
-        kernel_initializer (str): name of layer bias f the initializer for kernel parameters (weights)
-        bias_initializer (str): name of the initializer for bias
-        attn_kernel_initializer (str): name of the initializer for attention kernel
-        kernel_regularizer (str): name of regularizer to be applied to layer kernel. Must be a Keras regularizer.
-        bias_regularizer (str): name of regularizer to be applied to layer bias. Must be a Keras regularizer.
-        activity_regularizer (str): not used in the current implementation
-        kernel_constraint (str): constraint applied to layer's kernel
-        bias_constraint (str): constraint applied to layer's bias
+
+    Optional Args:
```

Comment:
All arguments optional or otherwise should be under `Args:` to follow the Google docstring styles. You can check they are rendered correctly by building the api docs see the info here:
https://github.com/stellargraph/stellargraph-ml-private-issues/blob/master/README.md